packages/editor/src/store/selectors.js

```diff
@@ -864,7 +864,11 @@ export function getSelectedBlockClientId( state ) {
  * @return {?Object} Selected block.
  */
 export function getSelectedBlock( state ) {
-	const clientId = getSelectedBlockClientId( state );
+	const { start, end } = state.blockSelection;
```

Comment:
I was thinking whether the following would be easier to follow:
```js
const { start: clientId, end } = state.blockSelection;

if ( ! clientId || clientId !== end ) {
    return null;
}

return getBlock( state, clientId );
```
What do you think?

Reply:
I think the current proposal is clearer, it has less decision trees, so it's easier to follow for the reader.